src/content/advanced/cluster-management/automatic-node-termination/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-linkTitle: Automatic termination of unhealthy nodes
+linkTitle: Automatic Node Termination
```

Comment:
Let's please not use Sentence Case. We have this rule also in our [style guide](https://intranet.giantswarm.io/docs/content/style-guide/#capitalization).

Reply:
hmm I will then change it back, but I remember in the past with were using that notation (English fellow was strong about it :D )